postprocessing/list_ncvars/list_ncvars.csh

```diff
@@ -1,5 +1,29 @@
 #!/bin/tcsh -f
 #
+#***********************************************************************
+#                   GNU Lesser General Public License
+#
+# This file is part of the GFDL FRE NetCDF tools package (FRE-NCTools).
+#
+# FRE-NCTools is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at
+# your option) any later version.
+#
+# FRE-NCTools is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with FRE-NCTools (LICENSE.md).  If not, see
+# <http://www.gnu.org/licenses/>.
+#***********************************************************************
+#
+# Copyright (C) NOAA Geophysical Fluid Dynamics Laboratory, 2000-2010
```

Comment:
We should standardize on a format for the copyright line.  No need to make changes to all the files now, but we should have a standard way of writing it.

Reply:
Part of the work is moved to this issue: https://github.com/NOAA-GFDL/FRE-NCtools/issues/46

Reply:
Part of the work is moved to this issue: https://github.com/NOAA-GFDL/FRE-NCtools/issues/46